pdr_backend/lake/etl.py

```diff
@@ -121,13 +129,80 @@ def do_bronze_step(self):
         end_ts = time.time_ns() / 1e9
         print(f"do_bronze_step - Completed in {end_ts - st_ts} sec.")
 
+    def _get_max_timestamp_values_from(
+        self, table_names: List, build_mode: bool = False
+    ) -> Dict:
+        """
+        @description
+            Get the max timestamp values from the tables
+
+        @arguments
+            table_names - The list of table names to get the max timestamp values from
+
+        @returns
+            The max timestamp values from the tables
+        """
+
+        pds = PersistentDataStore(self.ppss.lake_ss.parquet_dir)
+        max_timestamp_query = "SELECT MAX(timestamp) as max_timestamp FROM {}"
+        values = {}
+        for table_name in table_names:
+            calc_table_name = get_table_name(table_name, build_mode)
+            result = pds.query_data(max_timestamp_query.format(calc_table_name))
```

Comment:
To improve:
- Build querystr by iterating through tables
- Reduce query to 1 execution